js/admin.js

```diff
@@ -7,6 +7,15 @@
 			mode: 'shell'
 		} );
 
+	$('.ads-txt-rerun-check').click(function(){
```

Comment:
make this check callback function a standalone helper function, then call it here and also on page load.

Reply:
Taken care of in https://github.com/10up/ads-txt/pull/19/commits/5c1342d2a2caa25df88d894975c18a4c50ad88f9